src/pages/CreateSpot/CreateSpotsStep1.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback } from 'react';
+import { NavLink, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { PageWrapperSpaceBetween } from 'components/PageWrapper';
+import { InfoCard } from 'components/InfoCard/InfoCard';
+import { WideButton } from 'components/Button';
+import { Stepper } from 'components/Stepper';
+import { TopNav } from 'components/TopNav/TopNav';
+import { SpotAndPersonaProps } from 'components/CreateSpotAndPersona/SpotAndPersonaProps';
+import ContentImg from 'assets/ContentImg.svg';
+import TeamImg from 'assets/TeamImg.svg';
+import EyeImg from 'assets/EyeImg.svg';
+import SchemeImg from 'assets/SchemeImg.svg';
+import CustomImg from 'assets/CustomImg.svg';
+
+const InformationUnderText = styled(InfoCard)`
+  && .iTLFNI {
```

Comment:
what is this class?

Reply:
to change style in Info card

Reply:
are You sure this is not dynamic and while bulding the release it can change? @pumanitro ?

Reply:
@JanikOskar This is how styled-components works. Please add this to reading list:
https://styled-components.com/docs/basics
> No class name bugs: styled-components generates unique class names for your styles. You never have to worry about duplication, overlap or misspellings.
We can't do it in that way because styled-components will change that class name.